common/unsafe/src/main/java/org/apache/spark/unsafe/types/DateTimeConstants.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.unsafe.types;
```

Comment:
why it's in unsafe package? I'd expect it to be in `...catalyst.utils`

Reply:
For `CalendarInterval` references

Reply:
these constants are not only used by `CalendarInterval`, are they?

Reply:
```scala
org.apache.spark.unsafe.types.CalendarInterval
org.apache.spark.sql.catalyst.util.IntervalUtils/DateTimeUtils
```

Reply:
yea, so why we put constants in the unsafe package?

Reply:
We can define these Constants in CalendarInterval and DateTimeConstants?

Reply:
We can still put it in the unsafe module, let's change the package name to `...catalyst.utils`

Reply:
done